rr_supervisor/include/supervisor.hpp

```diff
@@ -0,0 +1,54 @@
+/** @file supervisor.hpp
+ *  @brief Supervisor prototypes
+ *  @author Waleed Ahmed(w29ahmed)
+ *  @competition IARRC 2018
+ */
+
+ // Include guard
+#ifndef SUPERVISOR_H
+#define SUPERVISOR_H
+
+#include <stdio.h>
+#include <string>
+
+#include <ros/ros.h>
+#include <std_msgs/Bool.h>
+#include <geometry_msgs/Twist.h>
+#include <std_srvs/Trigger.h>
+#include <std_srvs/Empty.h>
+
+
+class Supervisor
+{
+  public:
+    Supervisor();
+    // Callback methods
+    bool startRace(std_srvs::Empty::Request &req, std_srvs::Empty::Response &res);
+    bool countLap(std_srvs::Trigger::Request &req, std_srvs::Trigger::Response &res);
+    void trackSpeed(const geometry_msgs::TwistConstPtr& msg);
+    void finishRace();
```

Comment:
your code is really clean!
nit: the function naming doesn't follow the standard https://google.github.io/styleguide/cppguide.html#Function_Names